.env.example

```
@@ -0,0 +1,13 @@
+PORT=
+SMTP=
+
+#DB
+DB_USER=
+DB_PASSWORD=
+DB_NAME=
+
+#SMTP
+SMTP_HOST=
+SMTP_PORT=
+SMTP_USER=
+SMTP_PASSWORD=
```

```suggestion
PORT=

#DB
DB_USER=root
DB_PASSWORD=secret
DB_NAME=serlo

#SMTP
SMTP_HOST=localhost
SMTP_PORT=2500
SMTP_USER=test
SMTP_PASSWORD=test
```
I would populate it with the values from docker compose. 
Besides that, `SMTP` is not used so far.

---

Thanks for highlighting this before done will resolve it

---

Done